src/ImageSharp.Drawing/Pens/Pen{TPixel}.cs

```diff
@@ -26,16 +27,16 @@ namespace ImageSharp.Drawing.Pens
     public class Pen<TPixel> : IPen<TPixel>
         where TPixel : struct, IPixel<TPixel>
     {
-        private static readonly float[] EmptyPattern = new float[0];
-        private readonly float[] pattern;
+        private static readonly ReadOnlySpan<float> EmptyPattern = new float[0];
+        private readonly ReadOnlySpan<float> pattern;
```

Comment:
Span-:policeman: says:
[`Span<T>` is stack only!](https://github.com/dotnet/corefxlab/blob/master/docs/specs/span.md#spant-will-be-stack-only) Should never be used as member in classes!